nvme.c

```diff
@@ -3437,7 +3439,8 @@ int __id_ctrl(int argc, char **argv, struct command *cmd, struct plugin *plugin,
 	NVME_ARGS(opts,
 		  OPT_FLAG("vendor-specific", 'V', &cfg.vendor_specific, vendor_specific),
 		  OPT_FLAG("raw-binary",      'b', &cfg.raw_binary,      raw_identify),
-		  OPT_FLAG("human-readable",  'H', &cfg.human_readable,  human_readable_identify));
+		  OPT_FLAG("human-readable",  'H', &cfg.human_readable,  human_readable_identify),
+		  OPT_FLAG("fahrenheit",      'f', &cfg.fahrenheit,      fahrenheit));
```

Comment:
I think we should use the locale information and not adding an explicit command line option.